JustSaying/AwsTools/MessageHandling/MessageDispatcher.cs

```diff
@@ -47,7 +47,7 @@ public class MessageDispatcher
             _messageContextAccessor = messageContextAccessor;
         }
 
-        public async Task DispatchMessage(SQSMessage message, CancellationToken cancellationToken)
+        public async Task DispatchMessage(SQSMessage message, MessageBatchInflightTracker inflightTracker, CancellationToken cancellationToken)
```

Comment:
I see now why the tracker isn't `internal`, but now that's making me think this whole class should be internal.

It doesn't seem very useful to anything outside of JustSaying, so is probably only public for unit testing rather than using `InternalsVisibleTo`.

Reply:
You know I should just read all the comments first before responding to each of them. Yes I agree, it looked internal to me other than for tests.